front/src/modules/simulationResult/components/ChartHelpers/enableInteractivity.tsx

```diff
@@ -242,14 +252,14 @@ export const enableInteractivity = <
   chart: T | undefined,
   selectedTrainData: Data,
   keyValues: ChartAxes,
+  supplementaryValues: ChartAxes[], // more values to display on the same chart
```

Comment:
Can you move this argument so that it is the last of the list ? This way, you can set a default value
```suggestion
  additionalValues: ChartAxes[] = [], // more values to display on the same chart
```

Reply:
done